torch/csrc/jit/autodiff.cpp

```diff
@@ -334,17 +334,33 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
       }
       return {sizes.at(dim) > 1 ? grads.at(0) : grads.at(0).unsqueeze(dim), nullptr};
 
-    } else if (node->matches("aten::cat(Tensor[] tensors, int dim) -> Tensor", /*const=*/attr::dim)) {
+    } else if (
+        node->matches(
+            "aten::cat(Tensor[] tensors, int dim) -> Tensor",
+            /*const=*/attr::dim) ||
+        node->matches(
```

Comment:
Please add these schemas to https://github.com/pytorch/pytorch/blob/28e1571843f1b12e933e789ddcbf54339c92e24d/torch/csrc/jit/autodiff.cpp#L25. This enables autodiff testing for them.

Reply:
Actually please don't! We don't have the infrastructure we need to differentiate lists! This code can be simply removed, because it's dead.